apps/chameleon/app/(profile)/profile/edit/_components/profileSkill.tsx

```diff
@@ -0,0 +1,18 @@
+'use client'
+
+// import { useState } from 'react'
+import { Button } from '@/components/ui/button'
+
+interface ProfileSkillProps {
+  name: string
+}
+
+export default function ProfileSkill({ name }: ProfileSkillProps) {
```

Comment:
It's possible that since this is so minor, we might not need a file for it; it could be located on the same page.  I do see why you abstracted it, though.

Reply:
yea i was originally using it to track if it was clicked on or not to modify the styling with a useState. But now that i'm using a different approach of simply adding / removing skills instead, i agree with you.